expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -128,9 +128,9 @@ void shouldReturnExpandedTicketContainingTheOrganizationOfTheOwnersAffiliationAs
         Class<? extends TicketEntry> ticketType, PublicationStatus status) throws Exception {
         var publication = TicketTestUtils.createPersistedPublication(status, resourceService);
         var ticket = TicketTestUtils.createPersistedTicket(publication, ticketType, ticketService);
-        var userAffiliation = publication.getResourceOwner().getOwnerAffiliation();
+        ticket.setOwnerAffiliation(randomUri());
         var expandedTicket = (ExpandedTicket) expansionService.expandEntry(ticket);
-        assertThat(userAffiliation, is(equalTo(expandedTicket.getOrganization().id())));
+        assertThat(ticket.getOwnerAffiliation(), is(equalTo(expandedTicket.getOrganization().id())));
```

Comment:
I belive this should be reversed?
You just set the ownerAffiliation 2 lines above, so we know what it is
We except the getOrganization to be equal to the ownerAffiliation.